packages/features/vault/src/side-drawers/create-vault-position/create-vault-position.tsx

```diff
@@ -70,52 +73,71 @@ export const CreateVaultPosition = () => {
   );
 
   return (
-    <VaultSideDrawer
-      action={VAULT_ACTIONS.CREATE_VAULT_POSITION}
-      transactionData={undefined}
-      vaultAddress={vaultAddress}
-      canSubmit={canSubmit}
-      updatedVaultAccount={updatedVaultAccount}
-    >
-      <Maturities
-        maturityData={vaultMaturityData || []}
-        onSelect={(marketKey: string | null) => {
-          updateState({ selectedMarketKey: marketKey || '' });
-        }}
-        currentMarketKey={selectedMarketKey || ''}
-        inputLabel={messages[VAULT_ACTIONS.CREATE_VAULT_POSITION].maturity}
-      />
-      <WalletDepositInput
-        availableTokens={[primaryBorrowSymbol]}
-        selectedToken={primaryBorrowSymbol}
-        onChange={({ inputAmount, hasError }) => {
-          updateState({ depositAmount: inputAmount, hasError });
-        }}
-        inputLabel={messages[VAULT_ACTIONS.CREATE_VAULT_POSITION].depositAmount}
-        errorMsgOverride={inputErrorMsg}
-      />
-      <SliderInput
-        ref={sliderInputRef}
-        min={0}
-        max={maxLeverageRatio / RATE_PRECISION}
-        onChangeCommitted={(leverageRatio) =>
-          updateState({
-            leverageRatio: Math.floor(leverageRatio * RATE_PRECISION),
-          })
-        }
-        errorMsg={
-          leverageRatioError ||
-          (underMinAccountBorrow
-            ? Object.assign(messages.error.underMinBorrow, {
-                values: { minBorrowSize, borrowAmount },
+    <>
+      <SummaryWrapper>
```

Comment:
Do you have to include the mobile vault summary like this in all the different drawers?